frontend/src/app/workflows/workflows.component.ts

```diff
@@ -25,7 +25,8 @@ export class WorkflowsComponent implements OnInit {
   public generateResult!: GenerateResult;
 
   public currentExampleDesc: string = 'Select example';
-  public examplesList = ['Empty workflow', 'Data Modelling', 'Microservices', 'Pages'];
+  public examplesList = ['New Workflow', 'Data Modelling', 'Microservices', 'Pages'];
+  private examplesMap!: Map<string, string>;
```

Comment:
I wouldn't use a map here and just load the example just in time when it is selected. This does not scale well for a lot of examples due to all the requests.